deploy/olm-catalog/portworx/1.0.0/portworxoperator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,254 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-operator.v1.0.0
+  namespace: placeholder
+  annotations:
+    capabilities: Full Lifecycle
+    categories: "Storage"
+    description: Cloud native storage solution for production workloads
+    containerImage: portworx/openstorage-operator:1.0.0
+    repository: https://github.com/libopenstorage/operator
+    createdAt: 2019-04-28T08:00:00Z
+    support: Portworx, Inc
+    certified: "true"
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "core.libopenstorage.org/v1alpha1",
+          "kind": "StorageCluster",
+          "metadata": {
+            "name": "portworx",
+            "namespace": "kube-system"
+          },
+          "spec": {
```

Comment:
Also set internalKVDB to true?

Reply:
Had not added it because it is the default anyway. Will put it to make it explicit.